Model/ResourceModel/Job.php

```diff
@@ -0,0 +1,46 @@
+<?php
+namespace Algolia\AlgoliaSearch\Model\ResourceModel;
+
+
+class Job extends \Magento\Framework\Model\ResourceModel\Db\AbstractDb
+{
+
+    public function __construct(
+        \Magento\Framework\Model\ResourceModel\Db\Context $context
```

Comment:
Can you add `\Magento\Framework\Model\ResourceModel\Db\Context` to `use` and one-line it?